### PR TITLE
feat!: Update Sentry, remove sideEffects

### PR DESCRIPTION
**Pull request by camflan**
Apollo was directly requiring sentry which was also triggering a Sentry.init sideEffect. I've updated this to take in a Sentry hub to use

Sentry was firing off init sideEffects as well. I've updated to use Sentry 7.47+ to help with bundle sizes too

### CODE REVIEW

**Comment by mariusc23** [comment]
To do a breaking change you suffix with a `!`, so `fix!: update sentry` or `feat!: update sentry`

**Comment by camflan** [comment]
> To do a breaking change you suffix with a `!`, so `fix!: update sentry` or `feat!: update sentry`

ah, thanks!

File: packages/sentry/package.json [hunk 1]
```diff
@@ -29,7 +35,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "@sentry/browser": "^6.12.0",
-    "@sentry/node": "^6.12.0"
+    "@sentry/browser": "^6.12.0 || ^7.47.0",
+    "@sentry/node": "^6.12.0 || ^7.47.0"
```
Comment (mariusc23) [inline packages/sentry/package.json:39]: I think we could just update to the latest? Any reason to keep the old one around?

File: packages/apollo/package.json [hunk 2]
```diff
@@ -35,11 +35,11 @@
     "react": "^16.0.0 || ^17.0.0 || ^18.0.0"
   },
   "dependencies": {
-    "@uplift-ltd/constants": "^1.8.7",
-    "@uplift-ltd/sentry": "^4.1.5"
+    "@uplift-ltd/constants": "^1.8.7"
   },
   "devDependencies": {
     "@apollo/client": "^3.7.6",
+    "@sentry/types": "^7.5.0",
```
Comment (mariusc23) [inline packages/apollo/package.json:42]: Should this match the `^7.47.0` in the sentry package? Or they are not in sync?

Comment (camflan) [reply packages/apollo/package.json]: dang, good catch. twas a typo

File: packages/sentry/package.json (hunk 1 above)
Comment (camflan) [reply packages/sentry/package.json:39]: hmm, I thought I removed v6

Comment (camflan) [reply packages/sentry/package.json]: I missed the package.json but had committed the yarn.lock 🤦

**Review by marcelkooi — APPROVED** [review]

**Pull request merged**